karma.conf.js

```diff
@@ -1,51 +1,43 @@
+const webpackConfig = require("webpack-config-narazaka-ts-js").webTest;
+
 module.exports = (config) =>
   config.set({
-    basePath: '',
-    frameworks: ['detectBrowsers', 'phantomjs-shim', 'mocha-debug', 'mocha'],
-    files: [
-      'mock/browser-global-vars.js',
-      'node_modules/power-assert/build/power-assert.js',
-      'node_modules/shiorijk/lib/*.js',
-      'node_modules/shiori_converter/lib/*.js',
-      'src/**/*.js',
-      'test/**/*.js',
+    frameworks: ["detectBrowsers", "mocha"],
+    files:      [
+      "test/**/*.js",
+      "test/**/*.ts",
     ],
-    exclude: ['**/*.swp'],
+    exclude:       ["**/*.swp"],
     preprocessors: {
-      'src/**/*.js': ['babel', 'coverage'],
-      'test/**/*.js': ['babel'],
-    },
-    babelPreprocessor: {
-      options: {
-        presets: ['es2015'],
-        plugins: ['babel-plugin-espower'],
-        sourceMap: true,
-      },
-      filename: (file) => file.originalPath.replace(/\.js$/, '.es5.js'),
-      sourceFileName: (file) => file.originalPath,
+      "test/**/*.js": ["webpack"],
+      "test/**/*.ts": ["webpack"],
     },
+    webpack:          webpackConfig,
     coverageReporter: {
-      reporters: [{type: 'lcov'}],
-      instrumenters: {isparta: require('isparta')},
-      instrumenter: {
-        'src/**/*.js': 'isparta',
+      reporters: [
+        {type: "lcov"},
+        {type: "html"},
+        {type: "text"},
+      ],
+    },
+    reporters:      ["mocha-own", "coverage"],
+    detectBrowsers: {
+      postDetection(availableBrowsers) {
```

Comment:
__[jshint]__
*  'concise methods' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->